types/office-js-preview/index.d.ts

```diff
@@ -21185,6 +21264,16 @@ declare namespace Excel {
          * [Api set: ExcelApi 1.1]
          */
         readonly rowIndex: number;
+        /**
+         *
+         * Represents if ALL the cells would be saved as an array formula
+            Returns true if ALL cells would be saved as an array, or false if ALL cells would NOT be saved as an array formula.
+            Returns null if there are cells both which would be saved as an array formula and would not be
```

Comment:
This seems incomplete.

Also, won't this all end up being on one line in the docs?

Reply:
Sorry, but this statement is still not making sense to me...